src/hooks/useGetVp.ts

```diff
@@ -24,10 +24,8 @@ export const useGetVp = () => {
   strategies = st as unknown as Strategy[];
   const account = useAccount();
   const { data: vp } = useQqueryVp(proposal, account.address);
-  if (vp) {
-    return vp;
-  }
-  return 0;
+
+  return { data: vp, connected: account.isConnected };
```

Comment:
super minor, would probably call this isConnected. just like to keep all books as is / has. we can fix is some other time